examples/gitlab/index.js

```diff
@@ -0,0 +1,69 @@
+const express = require('express');
+const everyauth = require('@fusebit/everyauth-express');
```

Comment:
Pull this out into a space by itself, i.e.:
```
const express = require('express');
const { Gitlab } = require('@gitbeaker/node');
const { v4: uuidv4 } = require('uuid');
const cookieSession = require('cookie-session');

const everyauth = require('@fusebit/everyauth-express');

const app = express();
```
etc.  That visually highlights it.